deploy_function.yaml

```diff
@@ -0,0 +1,30 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: ((image_registry))/eq-cloud-function-deploy-image
```

Comment:
This is fine for now, and can be updated when https://trello.com/c/d9g6cH4C/4494-new-build-cloud-function-deploy-image-s merges